src/audio-worklet/design-pattern/wasm-supersaw/coi-serviceworker.js

```diff
@@ -0,0 +1,115 @@
+/*! coi-serviceworker v0.1.6 - Guido Zuidhof, licensed under MIT */
```

Comment:
I think we need this file under `src/third_party/` and all the examples should refer that single file.

Reply:
Ah, better approach! Thanks for pointing out. 
I will make the changes.